--- a/base/imagesize.py
+++ b/base/imagesize.py
@@ -183,10 +183,10 @@
              re.compile("^\xFF\xD8")                : ('image/jpeg', jpegsize),
              re.compile("^\x89PNG\x0d\x0a\x1a\x0a") : ('image/png', pngsize),
              re.compile("^P[1-7]")                  : ('image/x-portable-pixmap', ppmsize),
-             re.compile('\#define\s+\S+\s+\d+')     : ('image/x-xbitmap', xbmsize),
-             re.compile('\/\* XPM \*\/')            : ('image/x-xpixmap', xpmsize),
+             re.compile(r'#define\s+\S+\s+\d+')     : ('image/x-xbitmap', xbmsize),
+             re.compile(r'/\* XPM \*/')             : ('image/x-xpixmap', xpmsize),
              re.compile('^MM\x00\x2a')              : ('image/tiff', tiffsize),
-             re.compile('^II\*\x00')                : ('image/tiff', tiffsize),
+             re.compile('^II\\*\x00')               : ('image/tiff', tiffsize),
              re.compile('^BM')                      : ('image/x-bitmap', bmpsize),
              re.compile("^\x8aMNG\x0d\x0a\x1a\x0a") : ('image/png', pngsize),
            }
--- a/fax/ledmfax.py
+++ b/fax/ledmfax.py
@@ -43,7 +43,7 @@
 
 # **************************************************************************** #
 
-http_result_pat = re.compile(b"""HTTP/\d.\d\s(\d+)""", re.I)
+http_result_pat = re.compile(r"HTTP/\d.\d\s(\d+)", re.I)
 
 HTTP_OK = 200
 HTTP_ACCEPTED = 202
